src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/CodeGeneration/EventGenerator.cs

```diff
@@ -214,7 +214,7 @@ private static BlockSyntax GenerateBlock(IMethodSymbol accessor)
                 }
                 else
                 {
-                    AddAccessibilityModifiers(@event.DeclaredAccessibility, tokens, info, Accessibility.Private);
+                    CSharpCodeGenerationHelpers.AddAccessibilityModifiers(@event.DeclaredAccessibility, tokens, info, Accessibility.Private);
```

Comment:
had to qualify this as we have both a method and namespace with the same name in this layer.